pkg/actions/addon/delete.go

```diff
@@ -24,7 +22,7 @@ func (a *Manager) Delete(addon *api.Addon) error {
 	})
 
 	if err != nil {
-		if awsError, ok := err.(awserr.Error); ok && awsError.Code() == awseks.ErrCodeResourceNotFoundException {
```

Comment:
nit: we tend to use `awseks` as a way of avoiding the fact `eks` clashes with the local package we have `pkg/eks`, here they don't use both so its not a big deal, but thats why you will see `awseks` all over the codebase 😄

Reply:
good to know! thanks 😊